rpcs3/Emu/System.cpp

```diff
@@ -2561,9 +2588,15 @@ void Emulator::Kill(bool allow_autoexit, bool savestate)
 		m_config_mode = cfg_mode::custom;
 		read_used_savestate_versions();
 		m_savestate_extension_flags1 = {};
+
+		// Enable logging
+		rpcs3::utils::configure_logs(true);
```

Comment:
What about the other return?

Reply:
The other one is an error path, it should do nothing. configure_logs should have already been called there.